README.md

```diff
@@ -1,8 +1,8 @@
 # Sanctuary
 
-Sanctuary is a small functional programming library inspired by Haskell and
-PureScript. It depends on and works nicely with [Ramda][1]. Sanctuary makes
-it possible to write safe code without null checks.
+Sanctuary is a small functional programming library inspired by Haskell
+and PureScript. It depends on and works nicely with [Ramda][]. Sanctuary
```

Comment:
Markdown—or at least, [GitHub Flavored Markdown][1]—supports `[Foo][]` as shorthand for `[Foo][Foo]`. I like to use numeric reference links in most contexts, but for long documents such as this one updating the numbers each time a link is added or removed is irksome.


[1]: https://help.github.com/articles/github-flavored-markdown/